config.go

```diff
@@ -366,8 +358,17 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	walletConfig = &wallet.Config{
+		DataDir:				cfg.DataDir,
+		WalletPass:				cfg.WalletPass,
+		DisallowFree:			cfg.DisallowFree,
+		UseAddrIndex:			cfg.UseAddrIndex,
+		ChainParams:			activeNet.Params,
+		WalletDbName:			walletDbName,
```

Comment:
The db filename shouldn't be a config option.  Instead, move the constant from the old config.go to the new package.